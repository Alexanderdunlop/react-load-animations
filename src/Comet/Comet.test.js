import React from 'react'
import renderer from 'react-test-renderer'
import { Comet } from '..'

it('renders correctly', () => {
  const tree = renderer.create(<Comet />).toJSON()
  expect(tree).toMatchSnapshot()
})
