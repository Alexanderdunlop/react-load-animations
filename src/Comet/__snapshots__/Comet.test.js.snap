// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <style>
    [object Object]
  </style>,
  <div
    className="rla-comet"
  >
    <div />
    <div />
    <div />
  </div>,
]
`;
